### DISCUSSION TITLE
How to get badges from github ?

### DISCUSSION THREAD

**Discussion by james19978** [community/community#37733] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I'm new to github and I ended up finding this community amazing to solve my doubts, and I would like to know how I can get the new badges from github!

**Comment by Ezequiel-Mathias** [comment, marked as answer]
hello, welcome to our community, regarding emblems
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

Hope this helps !
